target/linux/realtek/files-5.15/arch/mips/rtl838x/prom.c

```diff
@@ -184,22 +184,18 @@ void __init prom_init(void)
 
 	pr_info("SoC Type: %s\n", get_system_type());
 
-	/* Early detection of CMP support */
-	if(soc_info.family == RTL9310_FAMILY_ID) {
-		mips_cpc_probe();
```

Comment:
I wonder if it makes sense to squash this and the previous commit. While the previous commit in itself makes sense, I think it falls perfectly into this commits category. Still, can we already do this for v5.10?